docs/source/sycamore/get_started.rst

```diff
@@ -22,24 +22,24 @@ Connectors for vector databases can be installed via extras. For example,
+By default, Sycamore works with Aryn DocParse to process documents. To run inference locally, install the `local-inference` extra as follows:
 
 .. code-block:: python
 
     pip install sycamore-ai[local-inference]
 
 ..
 
-Next, you can set the proper API keys for related services, like the Aryn Partitioning Service (APS) for processing PDFs (`sign-up here <https://www.aryn.ai/get-started>`_ for free) or OpenAI to use GPT with Sycamore's LLM-based transforms.
+Next, you can set the proper API keys for related services, like Aryn DocParse for processing documents (`sign-up here <https://www.aryn.ai/get-started>`_ for free) or OpenAI to use GPT with Sycamore's LLM-based transforms.
 
 Now, that you have installed Sycamore, you see it in action using the example Jupyter notebooks. Many of these examples load a vector database in the last step of the processing pipeline, but you can edit the notebook to write the data to a different target database or out to a file. `Visit the Sycamore GitHub <https://github.com/aryn-ai/sycamore/tree/main/notebooks>`_ for the sample notebooks.
 
 Here are a few good notebooks to start with:
 
-* An `intermediate ETL tutorial notebook <https://github.com/aryn-ai/sycamore/blob/main/notebooks/sycamore-tutorial-intermediate-etl.ipynb>`_ walking through an ETL flow with chunking (using APS), LLM-based data enrichment, data cleaning, and loading a Pinecone hybrid search index
-* A `notebook <https://github.com/aryn-ai/sycamore/blob/main/notebooks/tutorial.ipynb>`_ showing a simple processing job using APS to chunk PDFs, two LLM-based entity extraction transforms, and loading an OpenSearch hybrid index (vector + keyword)
-* A `notebook <https://github.com/aryn-ai/sycamore/blob/main/notebooks/VisualizePartitioner.ipynb>`_ that visually shows the bounding boxes created by the Aryn Partioning Service
-* A `more advanced Sycamore pipeline <https://github.com/aryn-ai/sycamore/blob/main/notebooks/metadata-extraction.ipynb>`_ that chunks PDFs using APS, does schema extraction and population using LLM transforms, data cleaning using Python, and loads an OpenSearch hybrid index (vector + keyword)
+* An `intermediate ETL tutorial notebook <https://github.com/aryn-ai/sycamore/blob/main/notebooks/sycamore-tutorial-intermediate-etl.ipynb>`_ walking through an ETL flow with chunking (using DocParse), LLM-based data enrichment, data cleaning, and loading a Pinecone hybrid search index
+* A `notebook <https://github.com/aryn-ai/sycamore/blob/main/notebooks/tutorial.ipynb>`_ showing a simple processing job using DocParse to chunk PDFs, two LLM-based entity extraction transforms, and loading an OpenSearch hybrid index (vector + keyword)
+* A `notebook <https://github.com/aryn-ai/sycamore/blob/main/notebooks/VisualizePartitioner.ipynb>`_ that visually shows the bounding boxes created by Aryn DocParse.
```

Comment:
nit: a bit weird that this bullet calls it 'Aryn DocParse' but other bullets here just use 'DocParse'